### DISCUSSION TITLE
[nexus2] Automatically drop repositories

### DISCUSSION THREAD

**Discussion by haydenbaker** [jreleaser/jreleaser#1601] [General]
We currently use the nexus2 deployer to release to sonatype (and then maven-central). [Here](https://github.com/smithy-lang/smithy/blob/main/build.gradle#L322-L335) is our current configuration.

In our previous (manual) process, we automatically drop our repositories when we release. Is this something we can currently accomplish through some undocumented configuration, or is this going to be a feature request? 

Thanks!

**Comment by aalmiray** [comment]
I'm afraid I don't understand what you need. Staging repositories are automatically "dropped" once released.

**Reply by haydenbaker** [reply]
The repository was marked as `released` in the UI, but stayed `released`. Are you saying it would've been dropped after some period of time?

**Reply by aalmiray** [reply]
Yes, that is my understanding.

**Reply by haydenbaker** [reply]
Sweet. I'll post back here once we execute another release and can confirm, thank you.

**Reply by aalmiray** [reply]
At least that's what happens on Maven Central OSSRH. Your local Nexus2 may be configured to retain staged repositories upon release. Check your settings.
